about.html

```diff
@@ -0,0 +1,247 @@
+            from 8. June to 7. July. <br>
+            After oniline public voting and Selection Committee's judging process. the logo from Naresh Agrawal won the
+            prize. <br>
+            After consultation with Mt. Agrawal we optimized the prize-winning-logo for this year's summit as following
+          </p>
+
+          <img class="cc-logo" src="img/about-logo.png" alt="cc common korea logo">
+
+        </div>
+
+      </article>
+
+
+      <article>
+
+        <div class="cc-past">
+          <h1 class="cc-past-heading">See the past CC Global Summits
+          </h1>
+          <div class="red-underline"></div>
+          <p class="cc-past-para about-desktop-display">Take a look at the last two CC Global Summits which took place
+            <br>
+            in Buenos Aires and in Warsaw.</p>
+
+          <p class="cc-past-para about-display">Take a look at the last two CC Global Summits which took place
+            in Buenos Aires and in Warsaw.</p>
+
+          <div class="cc-past-images">
+            <img src="img/about-2013.png" alt="past event 2013 image">
+            <img src="img/about-2011.png" alt="past event 2011 image">
+          </div>
+
+        </div>
+      </article>
+
+      <article>
+
+        <div class="partners-mobile-view about-desktop-display">
+
+
+          <h1 class="partners-heading-mobile-view">Partner</h1>
+          <div class="red-underline"></div>
+
+
+
+          <div class="partners-img-row1mobile-view">
+            <img src="img/partners/mozilla.svg" alt="mozilla logo partner">
+            <img src="img/partners/google.svg" alt="google logo partner">
+            <img src="img/partners/att-vector-logo.svg" alt="airbnb logo partner">
+
+          </div>
+
+          <div class="partners-img-row2-mobile-view">
+            <img src="img/partners/google.svg" alt="google partner logo">
+            <img src="img/partners/airbnb.svg" alt="airbnb partner logo">
+
+          </div>
+
+
+
+        </div>
+
+
+        <div class="partners-desktop about-display">
+
+
+          <h1 class="partners-heading-desktop">Partner</h1>
+          <div class="red-underline"></div>
+
+
+          <div class="partners-img-div">
+            <img src="img/partners/mozilla.svg" alt="mozilla partner logo">
+            <img src="img/partners/google.svg" alt="google partner logo">
+            <img src="img/partners/att-vector-logo.svg" alt="partner logo">
+
+
+            <img src="img/partners/google.svg" alt="google partner logo">
+            <img src="img/partners/airbnb.svg" alt="airbnb partner logo">
+          </div>
+
+
+
+
+
+
+
+        </div>
+      </article>
+
+
+    </section>
+
+    <footer>
+      <div class="footer-div">
+        <img src="img/partners/footer-logo.png" alt="cc logo">
+
+        <h1 class="footer-text"> 2015 Creative Commons Korea.<br>
+          Some Rights Reserved</h1>
+      </div>
+
+
+    </footer>
+
+  </div>
+
+
+  <script src="script.js"></script>
```

Comment:
- [ ] Good job with your about section :+1: But please use the same background color for the recent work section as defined in the original design and make your about section similar to the original design :smile: 

### Orignal design
![Screenshot from 2022-07-27 19-41-40](https://user-images.githubusercontent.com/6615394/181358728-12e551d4-0702-4690-98cd-8848c22a07d1.png)

### Your design
![Screenshot from 2022-10-20 17-35-02](https://user-images.githubusercontent.com/6615394/197019239-d9c3e9ff-630d-47dc-895f-1d6c2273af75.png)